taskRequests/details/style.css

```diff
@@ -216,6 +224,61 @@ body {
   color: #c3c3c3;
   font-weight: 600;
 }
+.reject__container {
+  text-align: center;
+}
+.requestors__container__reject__button {
+  padding: 0.375rem 0.5rem;
+  width: 80%;
+  background: #fff;
+  border: solid 1px var(--color-red-variant1);
+  font-weight: 700;
+  font-size: 1rem;
+  line-height: 1.5rem;
+  color: var(--color-red-variant1);
+  border-radius: 0.25rem;
+  cursor: pointer;
+}
+.requestors__container__reject__button:hover {
+  color: var(--color-white);
+  background: var(--color-red-variant1);
+  transition: 0.3s ease-in-out;
+}
+
+.circular-image {
+  border-radius: 50%;
+  max-width: 100%;
+  max-height: 100%;
+}
+
+.success {
+  color: var(--color-white);
+  background: var(--color-green);
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+}
+
+.failure {
+  color: var(--color-white);
+  background: var(--color-red-variant1);
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+}
```

Comment:
this should be a reusable class across the project in a global css file

Reply:
the elements on which these classes are used is defined in the index.html which is within `task\details` folder. if we move it inside the global css then it will not have access to the toast element defined inside `task\details` index. correct me if I am wrong.

Reply:
have we tried to pass the element reference to the function and check if that works